### PR TITLE
Virtual concierge page

### PR DESCRIPTION
**Pull request by shewood**
# Description

[VCON-225](https://decdvirtualconcierge.atlassian.net/browse/VCON-225)

Description of the task you completed and/or features added. If it is a visual component or feature,
add a screenshot(s) of it.
Added a folder for the projects
added a folder for virtual concierge
added an index page for the virtual concierge
added tests for the virtual concierge page and component
added images for the VC
added strings and placeholders for the translations. 

## Acceptance Criteria

Get the acceptance criteria for the story and/or task you are working on and put it all here.

## Help Requested
There seems to be an accessibility issue on the page but it appears that it is with other items on the page and not what I added and I am not sure where it is coming from exactly. 

## Checklist

- [ ] Strings use placeholders for translation (No hard coded strings)
- [ ] Unit tests have been added/updated

## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)

### CODE REVIEW

File: pages/projects/virtual-concierge/index.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+export default function Home(props) {
+  const { t } = useTranslation(["common", "vc"]);
+  const { asPath } = useRouter();
+
+  return (
+    <Layout
+      locale={props.locale}
+      langUrl={asPath}
+      // TODO not sure what the path is here
+      breadcrumbItems={[{ text: "Service Canada Labs", link: "/" }]}
```
Comment (nibivi77) [inline pages/projects/virtual-concierge/index.js:18]: The `link` should redirect to the `/home`.

**Review by nibivi77 — CHANGES_REQUESTED** [review]
The page is missing french images and french translations. Should this be a priority for this PR?

File: pages/projects/virtual-concierge/index.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+        {/*TODO these are the buttons on the bottom of the page that can be added if needed.  */}
+        {/* <div className="flex-col flex lg:flex lg:flex-row gap-1">
+          <TextButtonField href="/experiments" buttonText={t("experimentsBtnTxt")} idButton="ExperimentsButton" dataCyButton="ExperimentsButton" className="flex" />
+          <TextButtonField href="/about" buttonText={t("learnMoreAboutSCL")} idButton="AboutButton" dataCyButton="AboutButton" className="flex" />
+        </div> */}
+      </section>
+      <CallToAction
+        title={t("signupTitle")}
+        description={t("becomeAParticipantDescription")}
```
Comment (nibivi77) [inline pages/projects/virtual-concierge/index.js:116]: The `description` should be changed to `html` to render the unsubscribe link.
image

**Review by nibivi77 — CHANGES_REQUESTED** [review]

File: pages/projects/virtual-concierge/index.js (hunk 1 above)
Comment (DaceyTom2) [reply pages/projects/virtual-concierge/index.js]: Fixed

File: pages/projects/virtual-concierge/index.js (hunk 2 above)
Comment (DaceyTom2) [reply pages/projects/virtual-concierge/index.js]: changed description to html

**Comment by neilmispelaar** [comment]
I didn't have push access so had to create a separate PR #216 with the images for the VA

**Review by nibivi77 — APPROVED** [review]
This looks good! Thank you guys!

**Pull request merged**